tests/test_treelib.py

```diff
@@ -2,6 +2,8 @@
 # -*- coding: utf-8 -*-
 from __future__ import unicode_literals
 import sys
+import os
```

Comment:
nit: we shall try to order import alphabetically and as described in PEP8 - http://legacy.python.org/dev/peps/pep-0008/#imports